cdap-ui/app/cdap/components/PipelineScheduler/Store/index.js

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import {defaultAction, composeEnhancers} from 'services/helpers';
+import {createStore} from 'redux';
+import range from 'lodash/range';
+
+const INTERVAL_OPTIONS = {
+  '5MIN': '5min',
+  '10MIN': '10min',
+  '30MIN': '30min',
+  'HOURLY': 'Hourly',
+  'DAILY': 'Daily',
+  'WEEKLY': 'Weekly',
+  'MONTHLY': 'Monthly',
+  'YEARLY': 'Yearly',
+};
+
+const INTERVAL_OPTIONS_WITH_LABELS = [
```

Comment:
If we using the ids and labels just for UI why not merge these two? Seems a little redundant for us to have `INTERVAL_OPTIONS` and `INTERVAL_OPTIONS_WITH_LABELS`